jest.config.js

```diff
@@ -37,6 +37,12 @@ module.exports = {
     '!did/src/DemoKeystore/DemoKeystore.ts',
     '!did/src/DidDetails/DidDetails.utils.ts',
     '!did/src/DidDetails/FullDidDetails.utils.ts',
+    '!did/src/DidBatcher/DidBatchBuilder.ts',
```

Comment:
I have unit tested pretty much everything in this classes, but before of subclassing, jest expects me to re-test the same methods in the subclasses, resulting in a lower branch coverage %, of which I was really surprised. If you (reviewers) think the unit tests I have written are not extensive enough, I can work on improving them.

Reply:
you can try `/* istanbul ignore next */` as we discussed

Reply:
https://github.com/KILTprotocol/sdk-js/pull/473/commits/9162eee847803e1059b503e717ef117894944fc9